.circleci/config.yml

```
@@ -488,7 +488,11 @@ workflows:
       - all_circleci_checks_succeeded:
           requires:
             - e2e_tests
-            - win_e2e_tests
+            # win_e2e_tests-3.6 is not required due to the error
+            # `Cargo, the Rust package manager, is not installed or is not on PATH`.
+            # - win_e2e_tests
+            - win_e2e_tests-3.7
+            - win_e2e_tests-3.8
```

```suggestion
            - win_e2e_tests
```
since 3.6 won't be a problem in develop, and we do want to require the 3.9 tests.